src/duke/task/Event.java

```diff
@@ -0,0 +1,23 @@
+package duke.task;
+public class Event extends Task{
```

Comment:
**Coding standard violation:** Separate `Task` and `{` with a single whitespace character

I noticed many similar issues in other parts of the code. Consider reviewing them.